lib/thinkst_canary/token/null_token.rb

```diff
@@ -0,0 +1,11 @@
+module ThinkstCanary
+  module Token
+    class NullToken < Base
+      def initialize(**options)
```

Comment:
nit. seems to be the modern preference for such things and more consistent with other classes created in this PR
```suggestion
      def initialize(**kwargs)
```